src/utils/token-verifier.ts

```diff
@@ -213,9 +216,9 @@ export class FirebaseTokenVerifier {
           if (error) {
             if (error.name === 'TokenExpiredError') {
               const errorMessage = `${this.tokenInfo.jwtName} has expired. Get a fresh ${this.tokenInfo.shortName}` +
-                ` from your client app and try again (auth/${this.tokenInfo.expiredErrorCode.code}).` +
+                ` from your client app and try again (auth/${this.tokenInfo.errorCodeConfig.expiredError.code}).` +
```

Comment:
There's a reference to `auth` here. Should probably get rid of it.

Reply:
Oh good catch! I actually ended up finding more references to Auth. Ended up using generics in the verify token functions and moved the code to set the user id alias to one level up to `auth.ts`.
Let me know what you think. Thanks!

Reply:
Re: the error code prefix. I just removed the `auth` prefix, but the error code feels a bit redundant here as the error code will also be included in the public error type.

Reply:
This is something that will get cleaned up once we fix error handling for this SDK. For now this solution is adequate.